### PR TITLE
docs: update alert list visualization

### PR DESCRIPTION
**Pull request by mdcruz**
**What is this feature?**

Updates to the alert list visualization docs to include:
- how to configure an alert list visualization (video TBA)
- rearranging sections to match other visualization types

**NOTE**: Do not backport panel options shared file link to versions 10.3 and 10.4.

**Why do we need this feature?**

To explain to our users how to configure an alert list visualization easily and when to use it.

**Who is this feature for?**

Grafana OSS and Cloud users

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/alert-list/index.md [hunk 1]
```diff
@@ -49,18 +68,32 @@ Customize your visualization using the following settings.
...
-- **Alert instance label -** Filter alert instances using label querying, ex: `{severity="critical", instance=~"cluster-us-.+"}`.
-- **Folder -** Select a folder. Only alerts from dashboards in the folder selected will be displayed.
-- **Datasource -** Filter alerts from the selected data source.
+### Alert name
+
+Enter an alert name query.
+
+### Alert instance label
+
+Filter alert instances using [label](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/) querying, ex: `{severity="critical", instance=~"cluster-us-.+"}`.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/alert-list/index.md:86]:
```suggestion
Filter alert instances using [label](https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/annotation-label/) querying. For example,`{severity="critical", instance=~"cluster-us-.+"}`.
```

File: docs/sources/panels-visualizations/visualizations/alert-list/index.md [hunk 2]
```diff
@@ -49,18 +68,32 @@ Customize your visualization using the following settings.
...
+
+### Datasource
+
+Filter alerts from the selected data source.
+
+### Folder
+
+Select a folder. Only alerts from dashboards in the folder selected will be displayed.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/alert-list/index.md:94]:
```suggestion
Filter alerts by the selected folder. Only alerts from dashboards in this folder are displayed.
```

**Review by imatwawana — COMMENTED** [review]
Great update! I've made some structural and wording suggestions.

**Review by imatwawana — APPROVED** [review]
LGTM (pending addition of video)!

**Comment by mdcruz** [comment]
> LGTM (pending addition of video)!

@imatwawana I've added the video now, but the video is set to go live on the 22nd May :)

**Comment by mdcruz** [comment]
@imatwawana All ready to be merged. Video is now live :)

**Pull request merged**